### DISCUSSION TITLE
Useful currently unused dependencies

### DISCUSSION THREAD

**Discussion by je-cook** [Fusion-Power-Plant-Framework/bluemira#1069] [General]
The below dependencies we do not currently use but may be useful in future when adding new functionality

section-properties, structural analysis: https://github.com/robbievanleeuwen/section-properties 
fluids, Fluid dynamics component of Chemical Engineering Design Library: https://github.com/CalebBell/fluids
numdifftools,  differentiation problem solving tools: https://github.com/pbrod/numdifftools

**Comment by OceanNuclear** [comment]
For neutronics: we debated whether to use deterministic or stochastic modelling approaches. We went down the route of stochastic modelling because we were unaware of mature deterministic codes at the time. However, more recently, we have found out this one package which may be a mature deterministic code that could've done the same thing as OpenMC:
-  [THOR](https://github.com/NCSU-NCSG/THOR)

Edit 1: I did a bit more research and found two more useful code repo.
- [OpenNTP](https://github.com/mohamedlahdour/OpenNTP)
- [OpenMOC](https://github.com/mit-crpg/openmoc)

However, I must note that every repo on the list above is written in a mixture of python and Fortran/some other languages, and therefore are not pip-installable. They may be quite difficult to incorporate into our own repo.
